src/style.css

```diff
@@ -1,3 +1,86 @@
 body {
   margin: 0;
+  padding: 0;
+  display: block;
+}
+
+.header {
+  box-sizing: border-box;
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: #fff;
+}
+
+.logo_image {
+  width: 40px;
+  height: 40px;
+}
```

Comment:
Don't use tag names for styling (except `html` and `body`). Style elements using `.class`. Instead of `.logo_image`, which seems to be a class, use a more semantic class name related to its role or content.